### ISSUE TITLE
Subliminal's API docs are out of date

### ISSUE THREAD

**Issue by MaxGabriel** [inkling/Subliminal#185]
Subliminal's [API docs](http://inkling.github.io/Subliminal/Documentation/) are pretty out of date. Notably, the old README is still there, so you have to scroll down a bunch to even see the links to the API doc pages. There are some missing things like the `SLKeyboard` protocol as well.

I think this should be pretty easy—just build Subliminal's documentation target and copy the output to the gh-pages Documentation folder—but right now its deleting the generated output and installing directly to Xcode, and for the life of me I can't figure out which flag is doing that -.-

**Comment by wearhere** [comment]
Updated docs, ready to be uploaded for 1.1, are [here](https://github.com/inkling/Subliminal/pull/188).

Appledoc installs the docset by default (unless the `--no-install-docset` flag is specified), but docset and HTML generation aren't mutually exclusive. To keep the HTML, pass `--keep-intermediate-files`. The files will be dumped in `"${PROJECT_DIR}/Documentation/html"`.

Another modification to make to the Appledoc script for a "Release" build is to ignore Subliminal's [private headers](https://github.com/inkling/Subliminal/commit/67e8f7261a214ee8017037fe95a2e5de17345e71#diff-161fa2d1e0140f360679ead8a6c77ab1L19). That's not to be committed 'cause someone that builds the docs locally is probably a developer.

I am considering modifying _all_ doc builds' indexes, though. Currently we use Subliminal's README as index, but I'm not sure that's really relevant--I think it might be more appropriate to have maybe a small blurb about the framework at the top and then just links to the README and the Wiki for more information, keeping the focus on the API documentation. What do you think?

(Another reason to move away from using the README, btw, is that Appledoc chokes hardcore on GitHub-flavored markdown. Links and code blocks require a fair bit of manual cleanup IIRC.)

**Comment by MaxGabriel** [comment]
I like your idea of not using the README for the index file; good idea. Thanks for the help with appledoc

**Comment by wearhere** [comment]
I've finished fixing up the docs for 1.1 (and added a Rake task to make this a lot easier in the future) at #194. I decided to use the README after all to make the index a complete intro to Subliminal and for consistency with other projects i.e. on Cocoadocs, especially once I figured out how to prettify it like Cocoadocs does.

 I've uploaded the 1.1 `xar` (generated using that PR) branch already to test installation; the HTML docs will follow once we tag 1.1 ('cause we only host the latest HTML version).
